mdnsbridge/mdnsbridgeclient.py

```diff
@@ -110,7 +120,7 @@ def _createHref(self, service):
         port = service['port']
         return '{}://{}:{}'.format(proto, address, port)
 
-    def _updateServices(self, srv_type):
+    def updateServices(self, srv_type):
```

Comment:
Could this be changed back to an underscore? It doesn't seem like it will be necessary to call it externally.

Reply:
I am going to use this method in the node aggregator to flush the cache, so simplify the architecture